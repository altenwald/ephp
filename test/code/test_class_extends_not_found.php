<?php

class Noob 
      extends 
          Expert
{
}
