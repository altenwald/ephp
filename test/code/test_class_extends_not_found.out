
Fatal error: Class 'Expert' not found in {{CWD}}/test/code/test_class_extends_not_found.php on line 7
